### ISSUE TITLE
Logitech M705 - Arch - No active resolution

### ISSUE THREAD

**Issue by Utini2000** [libratbag/libratbag#826]
Hello everyone,

I am on Archlinux (with linux-hardend kenerl) with a Logitech M705 mouse.
Piper gives me a 404 error when running it.
So I decided to try ratbagctl, but I get errors there as well:

ratbagctl list
singing-gundi:       Logitech M705 

ratbagctl singing-gundi resolution active get
No active resolution. Please report this bug to the libratbag developers
0

ratbagctl singing-gundi rate get             
0

ratbagctl singing-gundi dpi get              
No active resolution. Please report this bug to the libratbag developers
0dpi

Any ideas what is going on? :(

**Comment by FFY00** [comment]
Yes, the M705 doesn't export the true DPI settings. It only allows you to configure a DPI multiplier, to be able to support this feature we would need to know the native resolution of the sensor the mouse uses.

You might want to try Solaar as it is more targeted to this line of devices.

**Comment by Utini2000** [comment]
I couldn't configure anything in piper. All settings were blanked out with 404. No buttons, no polling rate, nothing at all

**Comment by johnwilliamsdigital** [comment]
I'm on Kubuntu 19.10 and can't get the mouse to register at all, just get 404

no errors when running from the console. 

Advice?

**Comment by johnwilliamsdigital** [comment]
Ok, so I got ahold of an MX Master and a G602, and I'm having the same issue. 

What might I be doing wrong? lol.

**Comment by FFY00** [comment]
Nothing, the support is not great for those models, mainly because lack of time. Also,in the image the mouse is detected, we just don't have a image for it.

**Comment by johnwilliamsdigital** [comment]
ok thanks, any tips on another way to assign the thumb buttons?

**Comment by FFY00** [comment]
I am not sure what it going on there, AFAIK it should be working, there was probably a regression.
